daemon/container.go

```diff
@@ -1049,6 +1050,58 @@ func (container *Container) HostConfig() *runconfig.HostConfig {
 	return res
 }
 
+//Populates types.Container struct data
+func (c *Container) WriteAs(container *Container, config *ContainersConfig, newC *types.Container) {
```

Comment:
Why are we passing in *Container and *ContainersConfig ?  Can't we get both from 'c' ?